Update watchOS and visionOS to officially supported (#36219)
DataDog/documentation: content/en/real_user_monitoring/application_monitoring/ios/setup.mdoc.md

---

> title: iOS, tvOS, watchOS, and visionOS Monitoring Setup

Reviewer: ```suggestion
title: Apple Platform Monitoring Setup
```

> description: "Collect RUM data from your iOS and tvOS applications."

Reviewer: ```suggestion
description: "Collect RUM data from your Apple platform applications."
```

   text: "Learn about instrumenting SwiftUI applications"
 - link: "/real_user_monitoring/application_monitoring/ios/supported_versions"
   tag: "Documentation"

>    text: "RUM iOS, tvOS, watchOS, and visionOS monitoring supported versions"

Reviewer: ```suggestion
   text: "Apple platform monitoring supported versions"
```
